pages/blog/2022-summary.md

```diff
@@ -0,0 +1,545 @@
+
+### Workspace size
+
+Below you can see how many members joined our Slack every year. The numbers are pretty neat! People join Slack only if it is worth it, that they know:
+- someone will answer their call for help
+- it is a welcoming place for first-time contributors
+- we participate in many mentorship programs
+- this is the best place to get closer together with the community
+
+<Figure
+  src="/img/posts/2022-summary/slack-people-joining.webp"
+  widthClass="w-3/4"
+/>
+
+My prediction is that it will be growing and growing cause existing members will tell others what place it is and why it is worth joining.
+
+### Weekly active members
+
+Below I'm showing the average and median activity throughout the years. I think the median is more accurate as it eliminates super-high weeks (like typical conference-related noise) and weeks when we have lots of holidays simultaneously, like Christmas break.
+
+<Figure
+  src="/img/posts/2022-summary/slack-weekly.webp"
+  widthClass="w-3/4"
+/>
+
+Last year was the first time the median value was much lower than the average. It is higher than the previous year, but not something we got used to so far. We noticed a significant increase in Slack members, but I think we can say that this number doesn't correlate well with median activity.
+
+We should look into the data more deeply and understand what happened between April 23 and June 21 when numbers got much higher than usual. For example, what the heck happened on April 29 when we had 583 active members? Was AsyncAPI Mentorship the reason?
+
+Anyway, I definitely predict that data for 2023 will show an increase in activity as we plan to do a series of meetups around the world that will bring more and more people into Slack.
+
+### Channels popularity
+
+Below you can see all-time data that show the number of messages posted on each channel in Slack.
+
+The `General` channel is the oldest, and yeah, it is generally for general stuff... what a joker. Anyway, the most important is the discussion size in the `Tooling` channel. Specifications are boring, and people are interested more in how it works. On the other hand, tooling is most attractive to tech folks; coding and talking about it is more fun.   
+
+<Figure
+  src="/img/posts/2022-summary/slack-channel-usage.webp"
+  widthClass="w-3/4"
+/>
+
+Remember that AsyncAPI Initiative cares for both the specification and the tooling ecosystem.
+
+### New members' activity
+
+As I wrote in the intro, each community must be welcoming to new people and give them space inside so they feel comfortable interacting with others and feeling safe.
+
+Below you can see the top three members that joined the community in 2022 and were most active among other new joiners. Their activity was clearly reflected in their contributions. In the end, all the people you can see below are now members of the [AsyncAPI Technical Steering Committee](https://www.asyncapi.com/community/tsc).
```

Comment:
Not quite sure but I think `new-joiners` is an adjective and should be written together:

```suggestion
Below you can see the top three members that joined the community in 2022 and were most active among other new-joiners. Their activity was clearly reflected in their contributions. In the end, all the people you can see below are now members of the [AsyncAPI Technical Steering Committee](https://www.asyncapi.com/community/tsc).
```